### PR TITLE
Make notify section optional

### PR DESCRIPTION
**Pull request by okushnir-amplify**
Monitors without notification can be used for composite monitors. Let's allow them

### CODE REVIEW

File: data_kennel/config.py [hunk 1]
```diff
@@ -259,9 +260,6 @@ def get_monitors(self, tags=None):
                                                    notifications)
                     monitor['message'] = message
 
-                    # This isn't needed anymore
-                    del monitor['notify']
-
```
Comment (aoskotsky-amplify) [inline data_kennel/config.py]: Won't some monitoro still have a notify section?

Comment (okushnir-amplify) [reply data_kennel/config.py]: I saw the comment above, thought, it's not needed. Is it?

Comment (okushnir-amplify) [reply data_kennel/config.py]: Let's do it safe for now

Comment (jconstance-amplify) [reply data_kennel/config.py]: The comment doesn't mean that the del statement isn't needed, it means that the notify attribute is not needed anymore at that point. A bit of a difference. :D

**Review by jconstance-amplify — APPROVED** [review]

**Pull request merged**